docker/ci/install-build-essential.bash

```diff
@@ -15,4 +15,4 @@
 
 set -e
 
-apt-get install -y curl wget unzip git > /dev/null
+apt-get -qq install -y build-essential curl unzip > /dev/null
```

Comment:
In addition to `-qq`, we also need `> /dev/null` to silence the `apt-get` command.